src/tests/ftest/control/dmg_system_cleanup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+from command_utils import CommandFailure
+
+
+class DmgSystemCleanupTest(TestWithServers):
+    """Test Class Description:
+    This test covers the following requirement.
+    (SRS-326) Pool Management - automatic pool handle revocation
+
+    Test step:
+    1. Create 2 pools.
+    2. Create a container in each pool.
+    3. Call dmg system cleanup host.
+    4. Call daos pool list-cont on each pool. Each should fail with -1012.
```

Comment:
...Forever? If so that seems broken. Or is this a race we see if we query while the revocation is happening?

I don't think this is the right way to check if this is working. Just off the top of my head, it seems like we should:

- Open a few pool and container handles from different machines. Keep them open.
- Revoke the handles for one machine.
- Verify the machine's handles no longer work. (Try a pool query or something.)
- Verify the handles on the other machine still work.

Reply:
This test is completely rewritten so I'll upload the new single host test with these changes. I am still struggling to figure out how to do multi client tests in the framework.